packages/smooth_app/lib/pages/onboarding/knowledge_panel_page_template.dart

```diff
@@ -159,12 +159,32 @@ class _KnowledgePanelPageTemplateState
           margin: const EdgeInsetsDirectional.only(
             start: VERY_LARGE_SPACE,
             top: 10,
-            end: VERY_LARGE_SPACE,
+            end: 10,
             bottom: 10,
           ),
-          child: Text(
-            appLocalizations.hint_knowledge_panel_message,
-            style: TextStyle(color: Theme.of(context).cardColor),
+          child: Row(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: <Widget>[
+              Expanded(
+                child: Text(
+                  appLocalizations.hint_knowledge_panel_message,
+                  style: TextStyle(color: Theme.of(context).cardColor),
+                ),
+              ),
+              const SizedBox(width: VERY_LARGE_SPACE),
+              InkWell(
+                onTap: () {
+                  setState(() {
+                    _isHintDismissed = true;
+                  });
+                },
```

Comment:
We don't need that; there's already an `InkWell` on top, isn't there?

Reply:
@monsieurtanuki You're right! I have updated the code.